### PR TITLE
Add Host config field to QUICK-START

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Comment by alexjhawk** [comment]
This is a minor change, but I need at least one approval before merging.

**Review by oliver-walker-hms — APPROVED** [review]

File: QUICK-START.md
Comment (oliver-walker-hms) [inline QUICK-START.md:102]: whitespace change on last line of file

Comment (oliver-walker-hms) [reply QUICK-START.md:102]: I think a newline got added or deleted. just fix that and it's fine from me

Comment (alexjhawk) [reply QUICK-START.md:102]: Oh no - I just merged it before your comment.

Comment (alexjhawk) [reply QUICK-START.md:102]: I'm not sure why this change is present - I used the GitHub online editor.

Comment (oliver-walker-hms) [reply QUICK-START.md:102]: that's okay, it's very minor

**Pull request merged**
